plugins/textpattern.md

```diff
@@ -47,7 +47,7 @@ When using list commands make sure that the [lists plugin]({{ site.baseurl }}/pl
   {start: '####', format: 'h4'},
   {start: '#####', format: 'h5'},
   {start: '######', format: 'h6'},
-  {start: '1. ', cmd: 'InsertOrderedList'},
+  {start: '1. ', cmd: 'InsertOrderedList'}, //these configurations will need list plugin
```

Comment:
```suggestion
  {start: '1. ', cmd: 'InsertOrderedList'}, // these configurations will need the lists plugin
```

Reply:
Done.

Reply:
This hasn't been fixed yet sorry Wendel. The plugin name is still incorrectly `list` when it should be `lists`. It's' also missing `the` before the plugin name.

Reply:
Awesome, thanks. Done